### ISSUE TITLE
Switch net.train() to net.eval() during evaluation?

### ISSUE THREAD

**Issue by j96w** [milesial/Pytorch-UNet#31]
Since we are using BatchNorm2d in our U-net Model. Should we switch the network model to evaluation mode when we run net_eval()? Currently, it seems that this code does nothing about this. Should we add net.train() before training and net.eval() before evaluation?

**Comment by milesial** [comment]
You're right, I'm fixing it right now thank you.

**Comment by milesial** [comment]
Fixed in 3ab8272414, ab255c4 and 3af421f306. (Sorry for the separate commits, working on the web editor)
